Documentation/DebugMenu.md

```diff
@@ -27,7 +27,7 @@ This indicates the High Water mark for the stack for the PID thread. The smaller
 
 ### Time
 
-This just shows how many seconds the unit has been powered for.
+This just shows how many milliseconds the unit has been powered for.

```

Comment:
@Ralim
Hopefully I am right with this, please let me know, so I can take this PR down in time. 😁

regards